src/Models.fs

```diff
@@ -10,18 +10,29 @@ type Qty = int
 type Gem = Red | Green | Brown | White | Blue | Gold
 type Coin = | Gem of Gem
 
-type Card = {
-    image: Url;
-    cost: Gem list;
-    provides: Gem; 
-    victoryPoints: int;
-}
+/// count upward to victory
```

Comment:
triple slashes hooks into "doc comment" features without needing to add xml tags